api/python/t4/packages.py

```diff
@@ -238,9 +238,9 @@ def __call__(self):
 class Package(object):
     """ In-memory representation of a package """
 
-    def __init__(self, data=None, meta=None):
-        self._data = {} if data is None else data
-        self._meta = {'version': 'v0'} if meta is None else meta
+    def __init__(self, children=None, meta=None):
+        self._children = children or {}
+        self._meta = meta or {'version': 'v0'}
```

Comment:
Doesn't this mean that if the `meta` parameter is non-null, no `version` tag will be populated?

Reply:
Yeah... but I didn't change that - just made the code simpler. I'll undo it, though, just in case.